tests/test_device.py

```diff
@@ -1,26 +1,285 @@
 from datetime import datetime
 import pytest
 
+from pytradfri import error
 from pytradfri.const import (
-    ROOT_DEVICES, ATTR_NAME, ATTR_LIGHT_CONTROL, ATTR_LAST_SEEN)
+    ROOT_DEVICES,
+    ATTR_NAME,
+    ATTR_LIGHT_CONTROL,
+    ATTR_LAST_SEEN,
+    ATTR_LIGHT_COLOR_SATURATION,
+    ATTR_LIGHT_COLOR_HUE,
+    ATTR_LIGHT_DIMMER,
+    ATTR_TRANSITION_TIME,
+    ATTR_LIGHT_COLOR_X,
+    ATTR_LIGHT_COLOR_Y,
+    ATTR_LIGHT_MIREDS,
+    ATTR_LIGHT_STATE,
+    ATTR_LIGHT_COLOR_HEX
+)
 from pytradfri.device import Device
 from devices import (
-    LIGHT_W, LIGHT_WS, LIGHT_CWS, REMOTE_CONTROL, MOTION_SENSOR)
+    LIGHT_W, LIGHT_WS, LIGHT_CWS, REMOTE_CONTROL)
```

Comment:
Did you remove tests for the motion sensor alltogether?

Reply:
I don't think we really need to. There are two device classes (lamps & remotes). These classes are both tested and there are no device-specific cases for the motion sensors like there are for the lamps. The raw JSON outputs for these devices are identically structured.

Raw JSON outputs: https://gist.github.com/Lakitna/b6b041410431bcedff67a947640fb281
Note that the known issue with `3300` and `15009` is still present in this data. I'll fix that later :)

Reply:
But shouldn't we test that the REMOTE_CONTROL and MOTION_SENSOR are both compatible with the code? It won't increase coverage, but ensure that we don't accidentally remove something that creates problem downstream.

Reply:
Hmm you are right. Even though right now they identify identically, that might change in the future. I'll double up here :)

Reply:
@Lakitna thanks!